pages/options.coffee

```diff
@@ -127,6 +129,19 @@ class ExclusionRulesOption extends Option
   getPassKeys: (element) -> element.querySelector(".passKeys")
   getRemoveButton: (element) -> element.querySelector(".exclusionRemoveButton")
 
+  sortRules: ->
+    rules = @readValueFromElement()
+    for rule in rules
+      key = rule.pattern
+      key = RegExp.$1 if 0 == key.search /^[a-z?*]+:\/\/(.*)/
+      key = RegExp.$1 if 0 == key.search /^www\.(.*)/
+      rule.key = key
+
+    rules.sort (a,b) -> if a.key < b.key then -1 else if b.key < a.key then 1 else 0
+    @element.innerHTML = ""
+    @appendRule rule for rule in rules
+    @onUpdated()
```

Comment:
@gdh1995.  Thanks for your input.  Your point re. the popup page (above) is a great catch.

Yes, @philc.  Just "always sort" seems best.

@gdh1995.  Have you thought at all about what the comparison function should be?  Straight string comparison?  What's here (which is a bit of a hack)?  Something better?